### DISCUSSION TITLE
windows怎么部署Vanblog啊，求教

### DISCUSSION THREAD

**Discussion by zjzhaue** [Mereithhh/vanblog#119] [Q&A]
windows怎么部署Vanblog啊，求教

**Comment by RayWangQvQ** [comment]
装docker

**Comment by Mereithhh** [comment]
基本上要不开 wsl 装 docker，要不裸部署了

**Comment by adongteyin** [comment]
新手）需要买域名吗？别人怎么访问，这方面我不太理解。我以前也弄了一个Hexo静态博客部署在Github上，域名就可以访问。这个是否也一样呢？

**Reply by Mereithhh** [reply]
需要域名，需要自己的云服务器。。。我会尽快写一个从零开始的教程的。。。

**Reply by adongteyin** [reply]
感谢

**Comment by CornWorld** [comment]
@zjzhaue 现在还需要帮助吗在这一块
